### ISSUE TITLE
Add notebook/code to visualize results

### ISSUE THREAD

**Issue by ehoelzl** [mlbench/mlbench-results#2]
It would be nice to have a code/notebook that takes care of generating the graphs for each benchmark task.

This way, all graphs would look the same, and it would be easier to post results on a blog-post/mlbench website

**Comment by Panaetius** [comment]
We already have https://github.com/mlbench/mlbench-core/blob/develop/mlbench_core/cli/cli.py#L245, no?

**Comment by ehoelzl** [comment]
Yes but the problem is that this only works with a running cluster. It is not always the case that we have all runs on the same cluster.
